Tests/NIOExtrasTests/LineBasedFrameDecoderTest.swift

```diff
@@ -194,23 +194,22 @@ class LineBasedFrameDecoderTest: XCTestCase {
             buffer.writeString(string)
             return buffer
         }
-
+        
```

Comment:
Please suggest me a plugin to remove trailing whitespaces from file while saving the file.